ci/scripts/r_revdepcheck.sh

```diff
@@ -19,11 +19,40 @@
 set -ex
 
 : ${R_BIN:=R}
-
+: ${ARROW_REVDEP_WORKERS:=1)}
+: ${N_JOBS:=$(nproc)}
```

Comment:
Could you add a comment in here about what these control + why they are separate (presumably because we want to build with multiple cores, but we can't run targets with multiple workers, yeah?)?